From 2aca0207ac49cf811b278daf57906ec109b1f10b Mon Sep 17 00:00:00 2001
From: Amara Emerson <amara@apple.com>
Date: Tue, 14 Jul 2020 00:25:38 -0700
Subject: [PATCH] Revert "[compiler-rt] [test] Allow expanding lit
 substitutions recursively"

This reverts commit 8372d505082aceb38417e0b561cd32f2e227597b.

It broke the TestCases/Darwin/asan-symbolize-partial-report-with-module-map.cpp test on green dragon.

(cherry picked from commit eafeb8af34946306a7382fa3801cf6e39a1c7226)
---
 compiler-rt/test/lit.common.cfg.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/compiler-rt/test/lit.common.cfg.py b/compiler-rt/test/lit.common.cfg.py
index 9d0c214bd9a..98a2f3c03e6 100644
--- a/compiler-rt/test/lit.common.cfg.py
+++ b/compiler-rt/test/lit.common.cfg.py
@@ -23,9 +23,6 @@ else:
     # bash on Windows is usually very slow.
     execute_external = (not sys.platform in ['win32'])
 
-# Allow expanding substitutions that are based on other substitutions
-config.recursiveExpansionLimit = 10
-
 # Setup test format.
 config.test_format = lit.formats.ShTest(execute_external)
 if execute_external:
-- 
2.28.0.236.gb10cc79966-goog

